modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/data_set_display/management_bar/components/filters/AutocompleteFilter.js

```diff
@@ -71,24 +83,34 @@ function getOdataString(value, key, selectionType) {
 		? `${key}/any(x:${value
 				.map(
 					(v) =>
```

Comment:
Is this already  minified? 😄  I'm kidding. It's hard to me to understand what this whole block does at first sight, maybe improve variable naming could help?